account_invoice_merge/models/account_invoice.py

```diff
@@ -36,10 +24,11 @@ def _get_invoice_key_cols(self):
     @api.model
     def _get_invoice_line_key_cols(self):
         fields = [
-            'name', 'origin', 'discount', 'invoice_line_tax_id', 'price_unit',
+            'name', 'origin', 'discount', 'invoice_line_tax_ids', 'price_unit',
             'product_id', 'account_id', 'account_analytic_id',
+            'uom_id'
         ]
-        for field in ['analytics_id']:
```

Comment:
The 'analytics_id' field was added by <code>account_analytic_plans</code> base module. But this base module was removed in v9.